### DISCUSSION TITLE
日本通過經濟安保法，強化半導體等供應鏈

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#647] [General]
https://technews.tw/2022/05/20/japan-law-for-optimize-supply-chain/

日本首相岸田文雄政府看板政策之一的「[經濟安全保障推進法]，11 日在參議院全體會議表決通過，未來將強化半導體等供應鏈、防止尖端技術外流等，預計 2023 年起分階段施行。

日本經濟新聞報導，經濟安全保障推進法（簡稱經濟安保法）4大重點包括強化供應鏈、確保基礎設施安全、官民合力研發尖端技術，及禁止公開部分有危害國家安全之虞的專利。

**Comment by treeann** [comment]
供應鏈超厲害的拉 這種應用不學太可惜了

**Comment by Lai27330** [comment]
強化對整個供應鏈的掌握就能降地對各原物料進口國的依賴程度了。

**Comment by leon10456** [comment]
感謝樓主的分享

**Comment by stan890425** [comment]
這樣就不會時不時缺晶片了~~

**Comment by jcpan821** [comment]
希望台灣也能這樣

**Comment by alexthebigpp** [comment]
希望台灣能維持半導體霸主的地位

**Reply by jason79461385** [reply]
台積電股票可以持續上漲 真的很重要呢

**Comment by Ben6uo** [comment]
芯片對於科技的發展是極度重要的
